[17.0][MIG] partner_invoicing_mode_monthly: Migration to 17.0

traitrantrobz opened the pull request:
Depends on:
- account-invoicing/pull/1699

legalsylvain committed:
new module account_invoice_pricelist;

oca-transbot committed:
OCA Transbot updated translations from Transifex

hbrunn committed:
use pricelist when calculating prices

sergio-teruel committed:
account_invoice_pricelist: Migration to v9

oca-transbot committed:
OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

rvalyi committed:
account_invoice_pricelist: Migration to 10.0

oca-transbot committed:
OCA Transbot updated translations from Transifex

amcor committed:
account_invoice_pricelist: Migration to 11.0

oca-travis committed:
Update account_invoice_pricelist.pot

OCA-git-bot committed:
README.rst

rschnapka committed:
Translated using Weblate (German)

Currently translated at 100,0% (5 of 5 strings)

Translation: account-invoicing-11.0/account-invoicing-11.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-11-0/account-invoicing-11-0-account_invoice_pricelist/de/

OCA-git-bot committed:
README.rst

Nikul-Chaudhary committed:
[MIG] account_invoice_pricelist v11 to v12

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update account_invoice_pricelist.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_pricelist/

OCA-git-bot committed:
[ADD] icon.png

bealdav committed:
FIX acc_invoice_pricelist: tax include in multicompany

OCA-git-bot committed:
account_invoice_pricelist 12.0.1.0.1

bealdav committed:
FIX acc_invoice_pricelist: recompute tax after update pricelist

OCA-git-bot committed:
account_invoice_pricelist 12.0.1.0.2

am-technix committed:
Translated using Weblate (French)

Currently translated at 100.0% (5 of 5 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_pricelist/fr/

OCA-git-bot committed:
[UPD] README.rst

legalsylvain committed:
[FIX] bad display on invoices form

OCA-git-bot committed:
account_invoice_pricelist 12.0.1.0.3

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_pricelist/pt_BR/

Commit:
Added translation using Weblate (Spanish (Chile))

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (5 of 5 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_pricelist/es_CL/

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (5 of 5 strings)

Translation: account-invoicing-12.0/account-invoicing-12.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-12-0/account-invoicing-12-0-account_invoice_pricelist/hr/

manuelregidor committed:
[MIG] account_invoice_pricelist: black, isort, prettier

manuelregidor committed:
[MIG]account_invoice_pricelist: Migration to 13.0

manuelregidor committed:
[13.0][IMP] Supports multicurrency and different types of discounts

oca-travis committed:
[UPD] Update account_invoice_pricelist.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-13.0/account-invoicing-13.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-13-0/account-invoicing-13-0-account_invoice_pricelist/

Commit:
[14.0][MIG] account_invoice_pricelist: Migration to 14.0

kittiu committed:
[13.0][FIX] account_invoice_pricelist, fix test currency rate

OCA-git-bot committed:
account_invoice_pricelist 13.0.1.0.1

anasuarez1 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: account-invoicing-13.0/account-invoicing-13.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-13-0/account-invoicing-13-0-account_invoice_pricelist/es/

Zar21 committed:
[14.0][FIX] account_invoice_pricelist: Send invoice_date with context

The invoice date was not being sent to the function that looks for the tax that has to be applied and therefore it was looking for the current date

grindtildeath committed:
account_invoice_section_sale_order: Custom section name

Refactor preparation of sections

Refactor to allow different section grouping

Add field to allow different possibilities of grouping invoice lines from sale order

oca-travis committed:
[UPD] Update account_invoice_pricelist.pot

OCA-git-bot committed:
[UPD] README.rst

nicomacr committed:
[MIG] account_invoice_pricelist: Migration to 15.0

sebastienbeau committed:
[FIX] fix passing default_journal_id when invoicing

sebastienbeau committed:
[FIX] account_invoice_section_sale: fix invoice total when using currency

TDu committed:
[MIG] rename account_invoice_base_invoicing_mode

TDu committed:
[MIG] rename account_invoice_mode_monthly

StefanRijnhart committed:
[MIG] partner_invoicing_mode: Migration to 16.0

[UPD] Update partner_invoicing_mode.pot

[UPD] README.rst

partner_invoicing_mode 16.0.1.0.1

StefanRijnhart committed:
[IMP] partner_invoicing_mode_monthly: pre-commit stuff

StefanRijnhart committed:
[MIG] partner_invoicing_mode_monthly: Migration to 16.0

Commit:
[UPD] Update account_invoice_pricelist.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-15.0/account-invoicing-15.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-15-0/account-invoicing-15-0-account_invoice_pricelist/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-15.0/account-invoicing-15.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-15-0/account-invoicing-15-0-account_invoice_pricelist/

Commit:
[MIG] account_invoice_pricelist: Migration to 16.0

victorvermot committed:
add module block_payment corrected after Alexandre review

JaumeBforgeFlow committed:
[ADD] product_form_account_move_line_link

JaumeBforgeFlow committed:
[MIG] product_form_account_move_link: Migration to 14.0

victorvermot committed:
applied Alexandre corrections

pedrobaeza committed:
[FIX] account_invoice_pricelist: Make tests resilient

- Depending on the installed set of modules, the company currency may
  be USD or EUR. If the second case, these tests will fail, so we make
  sure that the company currency is USD for our tests, doing the change
  by SQL, as there's a Python constraint that prevents it.
- On contrary, the main pricelist doesn't change its currency, so we
  can have an incompatible currency.
- The onchange partner test should assert fixed data.

Not needed in v17 due to: https://github.com/odoo/odoo/pull/107113.

OCA-git-bot committed:
account_invoice_pricelist 15.0.1.0.1

JasminSForgeFlow committed:
[IMP] product_form_account_move_line_link: black, isort, prettier

JasminSForgeFlow committed:
[MIG] product_form_account_move_line_link: Migration to 15.0

pedrobaeza committed:
[FIX] account_invoice_section_sale_order: Avoid problems in integration tests

In OCA repo, now we have this error:

 Traceback (most recent call last):
  File "/__w/account-invoicing/account-invoicing/account_invoice_section_sale_order/tests/test_invoice_group_by_sale_order.py", line 109, in test_create_invoice
    self.assertEqual(line.name, result[line.sequence][0])
KeyError: 1

Commit:
[UPD] Update partner_invoicing_mode.pot

OCA-git-bot committed:
partner_invoicing_mode_monthly 16.0.1.0.1

RodrigoBM committed:
[REF] account_invoice_pricelist:
    * Better adaptation of the code to V16.
    * Bug fixes
    * Remove the price list in the supplier invoice.
    * Added access to the price list from the finance menu in the customer section.

Commit:
[UPD] Update account_invoice_pricelist.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-16.0/account-invoicing-16.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-account_invoice_pricelist/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-16.0/account-invoicing-16.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-account_invoice_pricelist/

Gelo-fl committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-account_invoice_pricelist/es/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (7 of 7 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-account_invoice_pricelist/pt_BR/

rousseldenis committed:
[REF] partner_invoicing_mode/monthly: Better extension mechanism

Allow to have a better extension mechanism when having multiple invoicing
modes.

rousseldenis committed:
[REF] partner_invoicing_mode/monthly: Better extension mechanism

Allow to have a better extension mechanism when having multiple invoicing
modes.

rousseldenis committed:
[IMP] partner_invoicing_mode: Partition invoicings

The field one_invoice_per_order wasn't used correctly.

So, now, create invoices using the partner option by calling
the grouping option on sale orders

Ivorra78 committed:
Added translation using Weblate (Spanish)

Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode/es/

[UPD] README.rst

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (14 of 14 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode_monthly
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode_monthly/es/

rousseldenis committed:
[IMP] partner_invoicing_mode: Add a field for next invoice date

rousseldenis committed:
[IMP] partner_invoicing_mode: Add the invoicing grouping on sale

To improve flexibility, the invoicing grouping option could be different
on the sale order level than on the partner level.

rousseldenis committed:
[IMP] partner_invoicing_mode: Add the invoicing grouping on sale

To improve flexibility, the invoicing grouping option could be different
on the sale order level than on the partner level.

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-16.0/account-invoicing-16.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-account_invoice_pricelist/

lucuswolfius committed:
[MIG] product_form_account_move_line_link: Migration to 16.0

rousseldenis committed:
[IMP] partner_invoicing_mode: Add some hooks

rousseldenis committed:
[IMP] partner_invoicing_mode: Don't loop search on sale orders

As the read_group() can aggregate sale order ids per partner,
use the ids list instead.

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

rousseldenis committed:
[IMP] partner_invoicing_mode: Don't set next_invoice_date as commercial field

As a company can have several different invoicing address, invoicing can occur at different
dates.

Ivorra78 committed:
Added translation using Weblate (Spanish)

mymage committed:
Added translation using Weblate (Italian)

Translated using Weblate (Italian)

Currently translated at 100.0% (8 of 8 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode_monthly
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode_monthly/it/

Commit:
[IMP] product_form_account_move_line_link: black, isort, prettier

Commit:
[17.0][MIG] product_form_account_move_line_link: Migration to 17.0

rousseldenis committed:
[IMP] partner_invoicing_mode: Update the invoice date just after creation

As validation jobs could last long time if there are plenty, ensure that
the wanted date is correctly set.

rousseldenis committed:
[IMP] partner_invoicing_mode: Use core grouping function in sale module

In order to be able to benefits from standard grouping function, override it
and add missing keys.

[UPD] Update partner_invoicing_mode.pot

[BOT] post-merge updates

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode/

Translated using Weblate (Italian)

Currently translated at 94.4% (17 of 18 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (18 of 18 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode/it/

Translated using Weblate (Spanish)

Currently translated at 100.0% (18 of 18 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode/es/

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode_monthly
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode_monthly/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode_monthly
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode_monthly/it/

berisoft-arg committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (14 of 14 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode_monthly
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode_monthly/es/

Commit:
[IMP] account_invoice_pricelist: pre-commit auto fixes

Commit:
[MIG] account_invoice_pricelist: Migration to 17.0

vvrossem committed:
[IMP] account_invoice_block_payment: black, isort, prettier

vvrossem committed:
[MIG] account_invoice_block_payment: Migration to 17.0

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (14 of 14 strings)

Translation: account-invoicing-16.0/account-invoicing-16.0-partner_invoicing_mode_monthly
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-16-0/account-invoicing-16-0-partner_invoicing_mode_monthly/hr/

traitrantrobz committed:
[IMP] partner_invoicing_mode_monthly: pre-commit auto fixes

NICO-SOLUTIONS committed:
[IMP] partner_invoicing_mode: pre-commit auto fixes

NICO-SOLUTIONS committed:
[MIG] partner_invoicing_mode: Migration to 17.0

nguyenminhchien reviewed (changes requested):
In 17.0, setting "Invoicing Mode" from partner_payment_mode is moved to the section "Customer Invoices". You should add a block "Monthly Invoicing Options" inside the setting "Invoicing Mode". From the image below, move 2 into 1 
![image](https://github.com/user-attachments/assets/e64e5ce5-054b-4da4-ade8-24feea105d79)

traitrantrobz committed:
[MIG] partner_invoicing_mode_monthly: Migration to 17.0

OCA-git-bot committed:
Merge PR #1676 into 17.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update product_form_account_move_line_link.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1733 into 17.0

Signed-off-by gurneyalex

Commit:
[UPD] Update account_invoice_block_payment.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-17.0/account-invoicing-17.0-account_invoice_block_payment
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-17-0/account-invoicing-17-0-account_invoice_block_payment/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: account-invoicing-17.0/account-invoicing-17.0-account_invoice_block_payment
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-17-0/account-invoicing-17-0-account_invoice_block_payment/it/

OCA-git-bot committed:
Merge PR #1699 into 17.0

Signed-off-by gurneyalex

Commit:
[UPD] Update partner_invoicing_mode.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1619 into 17.0

Signed-off-by yvaucher

Commit:
[UPD] Update account_invoice_pricelist.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-17.0/account-invoicing-17.0-partner_invoicing_mode
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-17-0/account-invoicing-17-0-partner_invoicing_mode/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-17.0/account-invoicing-17.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-17-0/account-invoicing-17-0-account_invoice_pricelist/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (18 of 18 strings)

Translation: account-invoicing-17.0/account-invoicing-17.0-partner_invoicing_mode
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-17-0/account-invoicing-17-0-partner_invoicing_mode/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (7 of 7 strings)

Translation: account-invoicing-17.0/account-invoicing-17.0-account_invoice_pricelist
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-17-0/account-invoicing-17-0-account_invoice_pricelist/it/

OCA-git-bot committed:
Merge PR #1804 into 17.0

Signed-off-by grindtildeath

Commit:
[UPD] Update account_invoice_section_sale_order.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-invoicing-17.0/account-invoicing-17.0-account_invoice_section_sale_order
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-17-0/account-invoicing-17-0-account_invoice_section_sale_order/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (18 of 18 strings)

Translation: account-invoicing-17.0/account-invoicing-17.0-account_invoice_section_sale_order
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-17-0/account-invoicing-17-0-account_invoice_section_sale_order/it/
